py-polars/polars/internals/frame.py

```diff
@@ -4072,11 +4072,11 @@ def n_unique(self) -> DataFrame:
         """
         return self._select_all().n_unique()
 
-    def quantile(self, quantile: float) -> DataFrame:
+    def quantile(self, quantile: float, interpolation: str = "nearest") -> DataFrame:
```

Comment:
Let's add a parameters sections which explains the possible interpolation strings